### PR TITLE
Remove strconv library dependency, use fmt instead

### PR DESCRIPTION
**Pull request by m-aravind11**
`fmt` makes the function more cleaner to read and understand

### CODE REVIEW

File: decimal/format.go [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+	"fmt"
 )
 
 func RoundAndFormat(val interface{}, places int32) (string, error) {
 	rounded, err := Round(val, places)
 	if err != nil {
 		return "", err
 	}
-	return strconv.FormatFloat(rounded, 'f', int(places), 64), nil
+	return fmt.Sprintf("%0.*f", places, rounded), nil
```
Comment (Siriayanur) [inline decimal/format.go:12]: requirement of padding leading zeroes?

Comment (m-aravind11) [reply decimal/format.go:12]: Yeah

Comment (m-aravind11) [reply decimal/format.go:12]: Can you refer the test cases?

Comment (Siriayanur) [reply decimal/format.go:12]: understood, we pad zeroes after decimal to fill defined space

**Review by Siriayanur — APPROVED** [review]

**Pull request merged**